MathlibTest/finset_builder.lean

```diff
@@ -62,8 +62,7 @@ example (a : α) : ({x ≥ a | p x} : Finset α) = (Ici a).filter p := rfl
 example (a : α) : ({x > a | p x} : Finset α) = (Ioi a).filter p := rfl
 
 -- elaborate as `Set` if the expected type is not `Finset ?α`
-example (a : α) : {x ≥ a | p x} = setOf fun x => x ≥ a ∧ p x := rfl
-example (a : α) : {x > a | p x} = setOf fun x => x > a ∧ p x := rfl
+example (a : α) : {x ≥ a | p x} = setOf fun x ↦ x ≥ a ∧ p x := rfl
+example (a : α) : {x > a | p x} = setOf fun x ↦ x > a ∧ p x := rfl
 
 end LocallyFiniteOrderTop
-
```

Comment:
This is the culprit.

Reply:
For the record: I'd land the test changes separately, and also change `Archive` and `Counterexamples` separately. This should shrink the size of this PR, and allow staggering dealing with conflicts.